src/firefly/js/ui/DownloadDialog.jsx

```diff
@@ -262,21 +262,10 @@ export function TitleField({style={}, value, label='Title:', size=30}) {
     );
 }
 
-export function ZipStructure({fieldKey='zipType'}) {
-    return (
-        <ListBoxInputField
-            fieldKey = {fieldKey}
-            initialState = {{
-                tooltip: 'File Structure',
-                label : 'File Structure:'
-            }}
-            options = {[
-                {label: 'Structured (with folders)', value: 'folder'},
-                {label: 'Flattened (no folders)', value: 'flat'}
-            ]}
-        />
-
-    );
+export function FileStructure({fieldKey='isFlattenedStructure'}) {
```

Comment:
if you take param zipType out from DonwloadDialog, you need to update app download processor in ife for ztf, sofia, wise